etc/env/base/contracts.toml

```diff
@@ -55,7 +55,7 @@ MAX_NUMBER_OF_HYPERCHAINS = 100
 L1_SHARED_BRIDGE_PROXY_ADDR = "0xFC073319977e314F251EAE6ae6bE76B0B3BAeeCF"
 L2_SHARED_BRIDGE_ADDR = "0xFC073319977e314F251EAE6ae6bE76B0B3BAeeCF"
 L2_SHARED_BRIDGE_IMPL_ADDR = "0xFC073319977e314F251EAE6ae6bE76B0B3BAeeCF"
-FRI_RECURSION_LEAF_LEVEL_VK_HASH = "0xcc4ac1853353538a166f5c2dde2c24e7e6c461dce8e3dc47d81e9139e1719456"
+FRI_RECURSION_LEAF_LEVEL_VK_HASH = "0xf9664f4324c1400fa5c3822d667f30e873f53f1b8033180cd15fe41c1e2355c6"
```

Comment:
please double check with @EmilLuta that it is fine to merge to main now

Reply:
I can also just revert